radio/src/targets/common/arm/stm32/usb_driver.h

```diff
@@ -29,6 +29,9 @@ enum usbMode {
   USB_JOYSTICK_MODE,
   USB_MASS_STORAGE_MODE,
   USB_SERIAL_MODE,
+//OW
+  USB_MAVLINK_MODE,
```

Comment:
There is no need for a special "USB MAVLink mode", this is plain USB CDC, nothing else. Please remove.